azure_functions_worker/dispatcher.py

```diff
@@ -146,7 +147,8 @@ async def dispatch_forever(self):
             root_logger = logging.getLogger()
 
             # Don't change this unless you read #780 and #745
-            root_logger.setLevel(logging.INFO)
+            root_logger.setLevel(logging.DEBUG if is_envvar_true(
+                PYTHON_ENABLE_DEBUG_LOGGING) else logging.INFO)
```

Comment:
I would recommend keeping the more common case as default. Also - it would be a little more readable to do something like the following:

```suggestion
            log_level = logging.INFO if not is_envvar_true(
                PYTHON_ENABLE_DEBUG_LOGGING) else logging.DEBUG
            root_logger.setLevel(log_level)
```